test/linters_test.go

```diff
@@ -97,6 +99,69 @@ func TestGciLocal(t *testing.T) {
 		ExpectHasIssue("testdata/gci/gci.go:7: File is not `gci`-ed")
 }
 
+func TestMultipleOutputs(t *testing.T) {
+	sourcePath := filepath.Join(testdataDir, "gci", "gci.go")
+	args := []string{
+		"--disable-all", "--print-issued-lines=false", "--print-linter-name=false", "--out-format=line-number,json:stdout",
+		sourcePath,
+	}
+	rc := extractRunContextFromComments(t, sourcePath)
+	args = append(args, rc.args...)
+
+	cfg, err := yaml.Marshal(rc.config)
+	require.NoError(t, err)
+
+	testshared.NewLintRunner(t).RunWithYamlConfig(string(cfg), args...).
+		ExpectHasIssue("testdata/gci/gci.go:7: File is not `gci`-ed").
+		ExpectOutputContains(`"Issues":[`)
+}
+
+func TestStderrOutput(t *testing.T) {
+	sourcePath := filepath.Join(testdataDir, "gci", "gci.go")
+	args := []string{
+		"--disable-all", "--print-issued-lines=false", "--print-linter-name=false", "--out-format=line-number,json:stderr",
+		sourcePath,
+	}
+	rc := extractRunContextFromComments(t, sourcePath)
+	args = append(args, rc.args...)
+
+	cfg, err := yaml.Marshal(rc.config)
+	require.NoError(t, err)
+
+	testshared.NewLintRunner(t).RunWithYamlConfig(string(cfg), args...).
+		ExpectHasIssue("testdata/gci/gci.go:7: File is not `gci`-ed").
+		ExpectOutputContains(`"Issues":[`)
+}
+
+func TestFileOutput(t *testing.T) {
+	f, err := os.CreateTemp("", "golangci_lint_test_result")
+	require.NoError(t, err)
+	f.Close()
+
+	resultPath := f.Name()
+	defer os.Remove(resultPath)
```

Comment:
Please `use t.TempDir`, it does not require removing and closing the file
```suggestion
	resultPath = t.TempDir() + "golangci_lint_test_result"
```

Reply:
changed, fixed also lint error